formtools/wizard/storage/base.py

```diff
@@ -43,15 +42,27 @@ def _get_current_step(self):
     def _set_current_step(self, step):
         self.data[self.step_key] = step
 
-    current_step = lazy_property(_get_current_step, _set_current_step)
```

Comment:
I'm not sure of the difference between the current implementation, the proposed implementation, and `property(_get_current_step, _set_current_step)` and all of them are passing the tests. Is there a test that could be added to show why the last version isn't appropriate?